numpy/lib/_function_base_impl.py

```diff
@@ -5198,8 +5198,10 @@ def delete(arr, obj, axis=None):
     ----------
     arr : array_like
         Input array.
-    obj : slice, int or array of ints
+    obj : slice, int, array-like of ints or bools
         Indicate indices of sub-arrays to remove along the specified axis.
+        Can be a single integer, a slice, an array of integers, a boolean array,
+        or any object that can be cast to an array of integers or booleans.
```

Comment:
That is quite a mouthful. What do we use in other places like this?

Reply:
Thanks for your suggestion, I deleted those two lines and only modified obj's description.